README.md

```diff
@@ -92,6 +92,20 @@ JSONSchemer.schema(
   # default: false
   insert_property_defaults: true,
 
+  # modify properties during validation. You can pass one Proc or a list of Procs to modify data.
+  # Proc/[Proc]
+  # default: nil
+  before_validation: ->(data, property, property_schema) {
```

Comment:
Maybe we should define these using `proc` to be consistent with the `ref_resolver` examples.